model/registry/user.yaml

```
@@ -28,8 +28,15 @@ groups:
         type: string
         stability: experimental
         brief: >
-          Unique identifier of the user.
+          Identifier of a user who interacts with a system. This identifier may be unique only through best-effort means and does not imply that the user is authenticated to the system.
+        note: >
+          It can be a random guid or a hash of the user's IP address. This is different from `user.hash` which is a hash of a known `user.id` or `user.name`.
+        examples: ['QdH5CAWJgqVT4rOr0qtumf']
+      - id: user.authentication.id
+        type: string
+        brief: "Unique identifier of an authenticated user in the system."
```

I really like `authentication` since we can now use it to describe auth attempt/process and not just the result

```suggestion
        brief: "Unique identifier of an authenticated user in the system."
        note: >
          The `user.authentication.id` MAY be used to identify a user attempting to authenticate if it's known at this stage.
```

does it align with your undestanding?